### ISSUE TITLE
old swift version

### ISSUE THREAD

**Issue by maup1** [pichillilorenzo/flutter_inappbrowser#4]
When I add the plugin to the pubspec.yaml I get an error: "The “Swift Language Version” (SWIFT_VERSION) build setting must be set to a supported value for targets which use Swift. Supported values are: 3.0, 4.0, 4.2. This setting can be set in the build settings editor."

When I open the project in Xcode I get: The target "flutter_inappbrowser" contains source code developed with Swift 2.x. Xcode 9 does not support building or migrating Swift 2.x targets. 

Any chance this plugin will be updated to the newest version?

**Comment by pichillilorenzo** [comment]
Wow it is really strange because I developed it using Swift 4! I will check about that, thanks.

**Comment by maup1** [comment]
I tried it but it still did not work. I search a little further and found this comments:

https://github.com/flutter/flutter/issues/13422#issuecomment-392133780

I created my project with "flutter create", in this case, the flutter_inappbrowser does not work because of the default option objc. After creating a project with "flutter create -i swift" it works!

I didn't know this exists, sorry for the unconvinced.
